README.md

```diff
@@ -13,3 +13,19 @@ Girl Develop It is a nonprofit organization that exists to provide affordable an
 ## Documentation
 
 We have migrated our documentation over to Read The Docs! Learn more about how to contribute to our code base by following along here: [http://gdi-new-site.readthedocs.org/](http://gdi-new-site.readthedocs.org/).
+
+###How to Vagrant
+_Note: This will move to the official documentation once the Vagrant dev environment is adopted into the staging branch._
+
+To  set up the vagrant environment locally:
+1. [Install VirtualBox](https://www.virtualbox.org/)
+2. [install Vagrant](https://www.vagrantup.com/downloads.html)
+3. [clone the git repository](https://github.com/girldevelopit/gdi-website.git)
+4. $ vagrant up
+5. $ vagrant ssh
+6. $ cd /opt/gdi/development
+7. $ bundle install
+8. $ rake db:create db:migrate db:seed
+9. $ bin/rails s
```

Comment:
I used the "rails s" command to get it running...so just nix bin (this is more of a note for when move this to actual documentation)